ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -0,0 +1,73 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "s3_log_system.h"
+
+#include <cstdarg>
+
+using namespace Aws::Utils;
+using namespace Aws::Utils::Logging;
+
+s3_log_system::s3_log_system(WT_EXTENSION_API *wtApi, uint32_t awsVerbose)
```

Comment:
How about we name the second parameter something like `wtVerbosityLevel`, to make it 100% clear that the constructor expects a wired tiger verbosity level as input. It would also be useful to add a comment explaining what we are doing with the mapping.